### PR TITLE
VVPSDK 1.0.0 release

### PR DESCRIPTION
**Pull request by AndriiMoskvin**
- [x] A lot of renamings :)
- [x] Version switched to 1.0.0 - but we still send 2.31 to microservice

### CODE REVIEW

**Comment by AndriiMoskvin** [comment]
Tutorials will be updated in separate PR.

File: support/VerizonVideoPartnerSDK-Info.plist [hunk 1]
```diff
@@ -19,7 +19,7 @@
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>2.31</string>
+	<string>1.0</string>
```
Comment (balavor) [inline support/VerizonVideoPartnerSDK-Info.plist:22]: This `CFBundleVersion` is the version that we send on our microservice. `CFBundleShortVersionString` is the SDK version.

File: VerizonVideoPartnerSDK.podspec [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  s.resources        = 'support/VerizonVideoPartnerSDK-Version.plist'
+  s.source_files     = 'sources/**/*.swift'
+  s.exclude_files    = 'sources/utils/Utils.swift', 
+                       'sources/utils/Recorder.swift', 
+                       'sources/utils/ActionComparator.swift', 
+                       'sources/**/*Test*', 
+                       'sources/**/Contents.swift'
+
+  s.ios.exclude_files  = 'sources/default controls'
+  s.tvos.exclude_files = 'sources/custom controls', 'sources/vpaid', 'sources/metrics/open measurement'
```
Comment (balavor) [inline VerizonVideoPartnerSDK.podspec]: Let's follow the previous style (s.exclude_files) and write these folder names from a new line.

**Review by balavor — DISMISSED** [review]

File: VerizonVideoPartnerSDK.podspec (hunk 2 above)
Comment (AndriiMoskvin) [reply VerizonVideoPartnerSDK.podspec]: 👍

File: support/VerizonVideoPartnerSDK-Info.plist (hunk 1 above)
Comment (AndriiMoskvin) [reply support/VerizonVideoPartnerSDK-Info.plist:22]: Discussed offline - will drop files - and introduce another way of getting VVP SDK version.

**Review by balavor — DISMISSED** [review]

**Comment by balavor** [comment]
You should fix project scheme and name in `Dangerfile`.

**Review by balavor — APPROVED** [review]

**Pull request merged**